src/pages/ListScholarships.jsx

```diff
@@ -57,6 +45,30 @@ function ListScholarships() {
     hideExpired: true,
   };
 
+  const filterChips = {};
+  if (Number.isInteger(minAmount)) {
+    filterChips[`Min $${minAmount}`] = () =>
+      setQueryParams({ minAmount: undefined });
+  }
+  if (Number.isInteger(maxAmount)) {
+    filterChips[`Max $${maxAmount}`] = () =>
+      setQueryParams({ maxAmount: undefined });
+  }
+  majors?.forEach(
+    (m) =>
+      (filterChips[m] = () =>
+        setQueryParams({
+          majors: majors?.filter((major) => major !== m),
+        }))
```

Comment:
The surrounding parentheses looks a bit hard to read IMO. What do you think of using curly braces instead?
```suggestion
          {
      filterChips[m] = () =>
        setQueryParams({
          majors: majors?.filter((major) => major !== m),
        })
      }
```